src/algorithm/retrieveWebidTrustedOidcIssuers.ts

```diff
@@ -1,48 +1,61 @@
-import { DataFactory } from "n3";
-import rdfDereferencer from "rdf-dereference";
-import type { Quad, Stream } from "rdf-js";
+// eslint-disable-next-line no-shadow
+import { URL } from "url";
+import type { Quad } from "@rdfjs/types";
+import { DataFactory, Parser, Store } from "n3";
+// eslint-disable-next-line no-shadow
+import fetch from "node-fetch";
 import { WebidDereferencingError } from "../error/WebidDereferencingError";
+import { WebidIriError } from "../error/WebidIriError";
 import { WebidParsingError } from "../error/WebidParsingError";
 import type { RetrieveOidcIssuersFunction } from "../type";
 
-const defaultGraph = DataFactory.defaultGraph();
-const oidcIssuer = DataFactory.namedNode(
-  "http://www.w3.org/ns/solid/terms#oidcIssuer"
-);
-
-async function dereferenceWebid(webid: string): Promise<Stream<Quad>> {
+async function dereferenceWebid(webid: string): Promise<string> {
   try {
-    return (await rdfDereferencer.dereference(webid)).quads;
+    const response = await fetch(webid, {
+      headers: {
+        // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Just lowercase it?

Reply:
Fixed in https://github.com/solid/access-token-verifier/pull/24/commits/57769c93f97c83a21edf089e4dc53aee3c1751c6